R/extractManualRecode.R

```diff
@@ -1,10 +1,22 @@
 #' Get manual recodes
 #'
-#' @param recodedList A data frame with the automatically recoded values.+
+#' @param recodedList A data frame with the automatically recoded values.
 #' @param varName Character string for the column containing the automatically recoded values.
-#' @return Returns a data frame which only includes the values that have to be recoded manually. It can be saved to excel for manual recoding.
+#' @return Returns a data frame which only includes the values that have to be recoded manually. It can be saved to Excel for manual recoding.
+#'
+#' @examples
+#' # example data frame
+#' df <- data.frame(id = 1:4,
```

Comment:
Please change the example data_frame a bit: 
- Build in some typos into the country-column, maybe also write Berlin instead of Germany.
- Name the `capital` column `capital_r`, and fill in some recoded values, e.G. `Germany` in the row containing `Berlin` in the `country` column. But also leave to rows with `NA`, showing they couldn't be recoded. 
I think this would be a more practical example, even though yours works fine as well.